pkg/models/devops/devops.go

```diff
@@ -162,8 +162,101 @@ func Validate(scmId string, req *http.Request) ([]byte, error) {
 	return resBody, err
 }
 
-func GetOrgSCM(scmId string, req *http.Request) ([]interface{}, error) {
-	baseUrl := fmt.Sprintf(JenkinsUrl+GetOrgSCMUrl+req.URL.RawQuery, scmId)
+func GetSCMOrg(scmId string, req *http.Request) ([]interface{}, error) {
```

Comment:
`GetSCMOrg` `GetSCMOrgRepo` `StopPipeline` `ReplayPipeline` shares a lot of code in common, can we extract that part into one function?